app/Models/Matter.php

```diff
@@ -72,6 +72,21 @@ public function client()
         return $this->hasOne(MatterActors::class)->whereRoleCode('CLI')->withDefault();
     }
 
+    public function sharedClient()
+    {
+        return $this->hasOne(MatterActors::class)->whereRoleCode('CLI')->whereShared(1)->withDefault();
+    }
+
+    public function payer()
+    {
+        return $this->hasOne(MatterActors::class)->whereRoleCode('PAY')->withDefault();
+    }
+
+    public function sharedPayer()
+    {
```

Comment:
The used role name in the table is "Payor", but "Payer" is also accepted. Why all these "sharedXXX" relationships?